speech_to_ra.py

```diff
@@ -67,29 +61,29 @@ def listen_and_return_text():
 
     try:            
             print("Recognizing...")
-            MyText = r.recognize_google(audio)
-            MyText = MyText.lower()
+            myText = r.recognize_google(audio)
+            myText = myText.lower()
 
-            print("English text heard: ",MyText)
-            return MyText
+            print("English text heard: ", myText)
+            return myText
 
     except sr.RequestError as e:
         print("Could not request results; {0}".format(e))
             
     except sr.UnknownValueError:
         print("unknown error occurred")
 
-def english_to_couriers_code(MyText):
+def english_to_couriers_code(text):
     ra_text = ""
-    for char in MyText:
-        if char in couriers_code:
-            ra_text += (couriers_code[char])
+    for char in text:
+        if char in COURIERS_CODE:
+            ra_text += COURIERS_CODE[char]
```

Comment:
this code is fine, but another way to do it in one line is
`return ''.join([COURIERS_CODE[char] if char in COURIERS_CODE else char for char in text])`

This might be less readable but it's less code. Up to you what matters more.